ocaml/xenopsd/xc/xenops_server_xen.ml

```diff
@@ -2523,6 +2525,9 @@ module VM = struct
                     )
                 then
                   raise (Xenopsd_error Failed_to_acknowledge_suspend_request) ;
+                with_tracing ~task ~name:"VM_migrate_downtime_begin" (fun () ->
```

Comment:
Why is it useful to trace a function that does nothing (`fun () -> ()`)? This does not define a useful span, as far as I can tell.

Reply:
They're each just creating a span to mark the time that the downtime ended/began. Ideally we would keep track of the downtime start and then create a single async span called VM_migrate_downtime with this start time and the downtime end time, but this proved quite difficult to do. I've created CP-54049 to do this but this is helpful enough for now as it saves having to look in the logs for the downtime start/end.